ui/src/app/pages/data/cohort-review/detail-header.component.tsx

```diff
@@ -514,8 +484,7 @@ export const DetailHeader = fp.flow(
             title='Go Back to the review set table'
             onClick={() => this.backToTable()}
           >
-            Back to review set
-            {serverConfigStore.get().config.enableMultiReview ? 's' : ''}
+            Back to review sets
```

Comment:
Just realized this change is causing the puppeteer failure for the `cohort-review.spec.ts` test. The change is correct but the test actually should have started failing when we first enabled the flag in the test env. It only passed because the strings 'Back to review set' and 's' were separated in the DOM.

![Screen Shot 2022-09-28 at 11 30 53 AM](https://user-images.githubusercontent.com/40036095/192835865-ecb15e71-815e-4f89-af21-26982c4f6614.png)

Need to add an 's' to the text at this line to fix the test: https://github.com/all-of-us/workbench/blob/main/e2e/app/text-labels.ts#L71

Reply:
done